src/main/Extensions/ExtensionLoader.ts

```diff
@@ -29,6 +30,7 @@ export class ExtensionLoader {
             new ColorConverterExtensionModule(),
             new CurrencyConversionModule(),
             new DeeplTranslatorModule(),
+            new VSCodeModule(),
```

Comment:
Please see that the items in this list are alphabetically ordered.

Reply:
Couldnt find an eslit rule for this 🥲